### PR TITLE
docker/ubu...: skip building C++/Java APIs

### PR DESCRIPTION
**Pull request by unclejack**
This reduces the size of the image from 13.6 GB to 10.9 GB.

before:
```
prod-contiv-ksr        0.0.1-1344-g8bf89ef           e18a3c80290f        About a minute ago   43.1MB
prod-contiv-cni        0.0.1-1344-g8bf89ef           cd733c5a936e        About a minute ago   18.9MB
dev-contiv-plugins     0.0.1-1344-g8bf89ef           f2b151059e39        About a minute ago   778MB
prod-contiv-cri        0.0.1-1344-g8bf89ef           bbb88aa32062        2 minutes ago        167MB
prod-contiv-vswitch    0.0.1-1344-g8bf89ef           04fa45ed1697        3 minutes ago        559MB
dev-contiv-vswitch     0.0.1-1344-g8bf89ef           e14c1a7428e4        4 minutes ago        13.6GB
dev-contiv-vswitch     0.0.1-1344-g8bf89ef-d95c39e   e14c1a7428e4        4 minutes ago        13.6GB
ligato/vpp-agent       pantheon-dev                  c01cf5ac082c        14 minutes ago       393MB
ligato/dev-vpp-agent   pantheon-dev                  20b99f21923c        27 hours ago         5.05GB
```
after:
```
prod-contiv-ksr        0.0.1-1344-g8bf89ef           6fcb4a28346d        6 minutes ago       43.1MB
prod-contiv-cni        0.0.1-1344-g8bf89ef           fb1e994c0887        6 minutes ago       18.9MB
dev-contiv-plugins     0.0.1-1344-g8bf89ef           83cd248e2aeb        6 minutes ago       778MB
prod-contiv-cri        0.0.1-1344-g8bf89ef           d3e0ebccbc64        7 minutes ago       167MB
prod-contiv-vswitch    0.0.1-1344-g8bf89ef           b939d40eb719        7 minutes ago       518MB
dev-contiv-vswitch     0.0.1-1344-g8bf89ef           bf12e3f3b808        7 minutes ago       10.9GB
dev-contiv-vswitch     0.0.1-1344-g8bf89ef-d95c39e   bf12e3f3b808        7 minutes ago       10.9GB
ligato/vpp-agent       pantheon-dev                  c01cf5ac082c        37 minutes ago      393MB
ligato/dev-vpp-agent   pantheon-dev                  20b99f21923c        27 hours ago        5.05GB
```

These changes are similar to https://github.com/contiv/docker-vpp/pull/14 and https://github.com/ligato/vpp-agent/blob/master/docker/dev_vpp_agent/Dockerfile#L43

### CODE REVIEW

File: docker/ubuntu-based/dev/Dockerfile [hunk 1]
```diff
@@ -30,9 +30,7 @@ RUN /bin/bash -c "if [ '${VPP_COMMIT_ID}' != 'xxx' ]; then \
         git checkout ${VPP_COMMIT_ID} && \
         rm -rf build-root/ && \
         git reset --hard HEAD && \
-        yes | make install-dep && \
-        make bootstrap && \
-        make pkg-deb; \
+        UNATTENDED=y make vpp_configure_args_vpp='--disable-japi --disable-vom' install-dep bootstrap pkg-deb; \
```
Comment (tiewei) [inline docker/ubuntu-based/dev/Dockerfile]: you you sure you can build contiv-vpp image with it ? i tried on docker-vpp and it doesn't work

Comment (unclejack) [reply docker/ubuntu-based/dev/Dockerfile]: What didn't work? I've tried these changes already.

Comment (tiewei) [reply docker/ubuntu-based/dev/Dockerfile]: when compile contiv-vpp it needs vpp headers, i saw things failed when added those `--disable-xx`
could you please try update `docker-vpp` and try compile `contiv-vpp` with https://github.com/contiv/vpp/tree/master/docker/vpp-vswitch ?

Comment (ondrej-fabry) [reply docker/ubuntu-based/dev/Dockerfile]: @tiewei I replied to the issue why it didn't work for you. You added extra parameter from what I initially suggested. https://github.com/contiv/docker-vpp/issues/3

**Pull request merged**